Project.toml

```diff
@@ -8,11 +8,18 @@ Automa = "67c07d97-cdcb-5c2c-af73-a7f9c32a568b"
 BGZFStreams = "28d598bf-9b8f-59f1-b38c-5a06b4a0f5e6"
 BioGenerics = "47718e42-2ac5-11e9-14af-e5595289c2ea"
 ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
```

Comment:
If FileIO is only used in the extension, I don't think we want it in the main deps sections